.github/workflows/code-quality.yml

```
@@ -0,0 +1,15 @@
+
+on:
+  push:
+    branches: [ main, development]
+  pull_request:
+    branches: [ main, development]
+
+jobs:
+  code-quality-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)